java/pom.xml

```diff
@@ -66,14 +66,14 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <!-- Define versions which are also used by grpc-client/pom.xml. -->
-    <grpc.version>1.16.0</grpc.version>
+    <grpc.version>1.33.0</grpc.version>
     <!-- NOTE Netty handler and boring SSL must be kept compatible with grpc
       https://github.com/grpc/grpc-java/blob/master/SECURITY.md#netty -->
     <netty.handler.version>4.1.48.Final</netty.handler.version>
     <tcnative.boring.ssl.version>2.0.30.Final</tcnative.boring.ssl.version>
 
-    <protobuf.java.version>3.6.1</protobuf.java.version>
-    <protobuf.protoc.version>3.6.1</protobuf.protoc.version>
+    <protobuf.java.version>3.11.4</protobuf.java.version>
+    <protobuf.protoc.version>3.11.4</protobuf.protoc.version>
```

Comment:
I _think_ this is fine for the time being. Before 1.15 lands I would like to update the golang side to similar versions of protoc and grpc